src/main/java/dk/bec/polonez/reservationsystem/repository/ReservationRepository.java

```diff
@@ -4,6 +4,10 @@
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
+
 @Repository
 public interface ReservationRepository extends JpaRepository<Reservation, Long> {
+
+    List<Reservation> findAllByDateFrom(long timestamp);
```

Comment:
**advice** Shouldn't timestamp by of some more meaningful type like ```LocalDateTime``` instead of ```long```?

Reply:
Hmm you're probably right, but it's too big of a change to implement now, because we have little time left.